picard/tagger.py

```diff
@@ -360,9 +388,57 @@ def load_to_picard(self, items):
             for item in parsed_items.mbids | parsed_items.urls:
                 thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
 
-        if parsed_items:
+        if parsed_items.non_executable_items():
             self.bring_tagger_front()
 
+    def get_album_pane_tracks(self):
+        tracks = set()
+        for album_name in self.albums:
+            for track in self.albums[album_name].iterfiles():
+                tracks.add(track)
+        return tracks
```

Comment:
Not sure the conversion to `set` is needed here, perhaps we can just use:

```python
def get_album_pane_tracks(self):
    for album in self.albums.values():
        for track in album.iterfiles():
            yield track
```

At worse, if required, conversion to set can be done in caller:

```python
for track in set(self.get_album_pane_tracks()):
```

What do you think?

Reply:
The reason I was adding them to a set was because I forgot/didn't know about the `values` method. So it is not required and the `yield` approach is much better, thanks :)